datasets/sk/rpvs/sk_rpvs.yml

```diff
@@ -31,10 +31,74 @@ publisher:
     people and organisations to use this data to build trust as well as
     reducing corruption and tax evasion.
 data:
-  url: https://s3.eu-west-1.amazonaws.com/oo-bodsdata/data/slovakia/json.zip
-  format: JSON+ZIP
+  url: https://rpvs.gov.sk/opendatav2/PartneriVerejnehoSektora?$skip=0
+  format: JSON
 
 lookups:
+  columns:
+    options:
+      - match: Id
+        value: id
+      - match: Meno
+        value: name
+      - match: Priezvisko
+        value: surname
+      - match: DatumNarodenia
+        value: dob
+      - match: TitulPred  # (e.g. academic or professional titles like Mgr., Dr., Ing.)
+        value: title_prefix
+      - match: TitulZa
+        value: title_suffix  # (e.g. PhD., CSc., MBA)
+      - match: ObchodneMeno
+        value: trading_name
+      - match: Ico
+        value: registration_number
+      - match: FormaOsoby
+        value: entity_type
+      - match: PlatnostOd
+        value: valid_from
+      - match: PlatnostDo
+        value: valid_to
+      - match: Adresa
+        value: address
+      - match: Partner
+        value: partner
+      - match: PravnaForma
+        value: legal_form
+      - match: MenoUlice
+        value: street_name
+      - match: OrientacneCislo
+        value: house_number
+      - match: Mesto
+        value: city
+      - match: MestoKod
+        value: city_code
+      - match: Psc
+        value: postal_code
+      - match: JeVerejnyCinitel
+        value: is_public_official
+      - match: KonecniUzivateliaVyhod
+        value: beneficial_owners
+      - match: VerejniFunkcionari
+        value: public_officials
+      - match: StatistickyKod
+        value: economic_classification
+      - match: "@odata.context"
+        value: "@odata.context"
+      - match: SupisneCislo
+        value: reference_number
+      - match: Identifikator
+        value: identifier
+      - match: CisloVlozky
+        value: registry_entry_number
+      - match: Vymaz
+        value: deletion_date
```

Comment:
Where do the last four fields occur and why don't we catch them in our context.audit? Maybe add a quick comment (if we know what they are)?